src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/ExtractOriginalAlignmentRecordsByNameSpark.java

```diff
@@ -21,22 +21,46 @@
 import java.util.stream.Collectors;
 
 /**
- * Created by shuang on 3/3/17.
+ * Find reads by name.
+ *
+ * <p>Reads a file of read (i.e., template) names, and searches an input file of reads to find names that match.
```

Comment:
I originally forgot to add "unlike PrintReadsSpark, this tool does not require reads to be coordinate-sorted"